tests_data/category_dropdown_without_naming.csv

```diff
@@ -0,0 +1,4 @@
+ru;Любой;https://auto.ria.com/search/?indexName=auto,order_auto,newauto_search
```

Comment:
remove path of url  **https://auto.ria.com**

Reply:
Done